terralab/logic/pipelines_logic.py

```diff
@@ -53,36 +53,28 @@ def validate_pipeline_inputs(pipeline_name: str, version, inputs_dict: dict):
         expected_input_keys.append(input_name)
```

Comment:
i know this wasnt changed here but do we need to check whether the `PipelineUserProvidedInputDefinition` is required from the user or not?  I dont remember if we only return requited inputs from the API (i would hope not).

Reply:
hmm good point, we should check this... we do return all user-provided inputs, whether required or not (and imputation just doesn't have any optional user-provided inputs), so we should do this check